inc/osvr/Util/ResetPointerList.h

```diff
@@ -44,7 +44,7 @@ namespace util {
         typedef typename RangeType::type IteratorType;
         typedef typename IteratorType::value_type PointerType;
         for (auto &ptr : range) {
-            ptr.reset()
```

Comment:
It isn't; I checked. I figured I'd fix it anyway. I wasn't sure if you have future plans for it or not.